config/components/rbac/role_binding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: manager-rolebinding
+  name: rolebinding
```

Comment:
I wonder if it makes any sense to suffix objects' names with their kinds, we typically do it for RBAC only. But that's not only we who did it:
![Screenshot_20201201_121849](https://user-images.githubusercontent.com/5887312/100727819-67580280-33cf-11eb-8e09-c546c7d7ec46.png)
So, I'm OK with leaving as is

Reply:
The only reason I do it is because `devworkspace-controller-` is the prefix, and has to prefix something as far as I know :)

I'd love to find an alternative -- the main detail that's important for kustomize is that the name applied to the unpatched yamls can be referenced elsewhere (so we can e.g. get the serviceaccount named "serviceaccount" even though it gets prefixed).